src/main/groovy/jp/co/soramitsu/devops/docker/DockerPlugin.groovy

```diff
@@ -293,6 +295,8 @@ class DockerPlugin implements Plugin<Project> {
             return 'openjdk:13'
         } else if (javaVersion == 17) {
             return 'openjdk:17-jdk-slim'
+        } else if (javaVersion == 21) {
+            return 'openjdk:21-jdk-slim'
```

Comment:
amazon images?

Reply:
I'd suggest unusual images to be set on the project level, not in the plugin

Reply:
why not jre?

Reply:
Tested:
eclipse-temurin:17-jre-alpine,
eclipse-temurin:21-jre-alpine.